src/linear_algebra/basic.lean

```diff
@@ -6,6 +6,7 @@ Authors: Johannes Hölzl, Mario Carneiro, Kevin Buzzard, Yury Kudryashov
 import algebra.big_operators.pi
 import algebra.module.pi
 import algebra.module.prod
+import algebra.module.submodule
```

Comment:
This is the only place in mathlib that used to get `submodule` from `algebra.module.basic` and now needs to be changed? I'm a bit surprised about that but I guess `linear_algebra.basic` is imported by `ring_theory.ideal` which provides submodules to most of the rest of mathlib.